modules/localgov_netcall_ai_widget/README.md

```diff
@@ -0,0 +1,30 @@
+## INTRODUCTION
+
+The LocalGov Netcall AI Widget module is a DESCRIBE_THE_MODULE_HERE.
```

Comment:
Err.. quite a few placeholders in this file.  If you can please look into them.

Reply:
Ooops, thanks Adnan.